packages/thirdweb/src/transaction/actions/to-serializable-transaction.ts

```diff
@@ -6,17 +6,44 @@ import type { PreparedTransaction } from "../prepare-transaction.js";
 import { encode } from "./encode.js";
 import { estimateGas } from "./estimate-gas.js";
 
-export type ToSerialiableTransactionOptions = {
+export type ToSerializableTransactionOptions = {
+  /**
+   * The transaction to convert to a serializable transaction.
+   */
   // biome-ignore lint/suspicious/noExplicitAny: TODO: fix later
   transaction: PreparedTransaction<any>;
+  /**
+   * The from address to use for gas estimation.
+   */
   from?: string;
 };
 
 /**
- * @internal
+ * Converts a prepared transaction to a serializable transaction.
+ * @param options - The transaction and additional options for conversion
+ * @returns A serializable transaction for inspection or submission to an account.
+ *
+ * @note For easier transaction sending, {@see sendTransaction}
+ * @example
+ * ```ts
+ * import { prepareTransaction, toSerializableTransaction } from "thirdweb";
+ *
+ * const transaction = await prepareTransaction({
+ *   transaction: {
+ *     to: "0x...",
+ *     value: 100,
+ *   },
+ * });
+ * const serializableTransaction = await toSerializableTransaction({
+ *   transaction,
+ * });
+ *
+ * account.sendTransaction(serializableTransaction);
+ * ```
+ * @transaction
  */
 export async function toSerializableTransaction(
-  options: ToSerialiableTransactionOptions,
+  options: ToSerializableTransactionOptions,
```

Comment:
wonder if we should rename this to `populateTransaction` ? kind ethers-y but it kinda does make sense since that's what its doing, filling in the blanks. wdyt?

Reply:
Works for me 👍 
Going to keep the internal name `toSerializable` since it matches the viem flow we're using and will export as `populateTransaction` for anyone looking to get the final tx params

Reply:
nah we should keep the same name, its gonna look messed up in the docs otherwise I think. lets just keep toSerializableTransaction - i'll make the change for you